aapl_IEX_model.ipynb

```diff
@@ -0,0 +1,1659 @@
+{
```

Comment:
<p>rename <code>roc</code>  to something more readable. probably <code>rate_of_change</code>  :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alongouldman/stock_market_neural_nets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>